### PR TITLE
Prepare release 3.0.0

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [x] Updated modules to prepare for release
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core pipelines lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by Lucpen** [comment]
Once this one is merged I have to try running a couple of cases and then make the release, I have opted for 3.0.0 because previous param files will no longer be valid after the last nf-tools update

**Comment by Jakob37** [comment]
I'll test running through a sample!

File: subworkflows/local/call_variants.nf [hunk 1]
```diff
@@ -81,10 +81,11 @@ workflow CALL_VARIANTS {
         ch_bed_in_annot = ADD_VARCALLER_TO_BED.out.gz.collect()
         ch_tbi_in_annot = ADD_VARCALLER_TO_BED.out.tbi.collect()
 
+        bed_with_tbi = ch_bed_in_annot.join(ch_tbi_in_annot)
+        in_bcftools_annot = REMOVE_DUPLICATES.out.vcf.join(REMOVE_DUPLICATES.out.tbi).join(bed_with_tbi)
```
Comment (fellen31) [inline subworkflows/local/call_variants.nf:85]: Would be easier (for me) to follow. 

```suggestion
        in_bcftools_annot = REMOVE_DUPLICATES.out.vcf
            .join(REMOVE_DUPLICATES.out.tbi)
            .join(ch_bed_in_annot)
            .join(ch_tbi_in_annot)
```

**Review by fellen31 — APPROVED** [review]
Nice! 👍 Could salmon quant and bcftools mpileup be updated on nf-core as well? Let me know if you need a review.

**Comment by Jakob37** [comment]
Hmm. I am running into the same issue as before with invalid FRASER counts 🤔 I thought this was working before.

This is from `k_j_counts.tsv.gz`

```
seqnames        start   end     strand  as.array(x, drop = TRUE)
chr1    12228   12612   +       1
chr1    12722   13220   +       1
chr1    827776  829002  +       5
chr1    829105  841199  +       2
chr1    847807  851926  +       4
chr1    829105  847653  +       7
chr1    855122  856448  +       4
chr1    850352  851926  +       1
chr1    827850  829002  +       2
chr1    861274  865259  +       1
chr1    961553  961628  +       2
```

Will debug this a bit more. Will let you know if I find that this is a Tomte error (rather than just an error on my part).

**Comment by Jakob37** [comment]
OK, with updated references this went through nicely! So looks like this version runs just fine. 

Not sure what is up with the FRASER count files. Will keep my eyes on if it comes up again 👀

**Pull request merged**